airbyte-webapp/src/pages/OnboardingPage/components/VideoItem/components/ShowVideo.tsx

```diff
@@ -25,7 +25,7 @@ const CloseButton = styled(Button)`
 
 const ShowVideo: React.FC<ShowVideoProps> = ({ videoId, onClose }) => {
   return (
-    <Modal onClose={onClose} clear closeOnBackground>
+    <Modal onClose={onClose} cardless>
       <CloseButton onClick={onClose} iconOnly>
```

Comment:
because this closes when a user touches the background already... I'm not sure that we need or want the "X" button here?

Reply:
Is this a UI/UX type question? Because I feel like an X doesn't hurt.

Reply:
Either.  I don't mind keeping it per se, it just doesn't serve any actual purpose (if users click anywhere outside the video it closes)

Reply:
I'd prefer if we could keep it. I think (unvalidated), the backdrop won't be focusable and thus not usable via keyboard only? Also it feels UX wise better to also have an explicit way that users will understand, they might simply not see otherwise that clicking on the backdrop will close it.

Reply:
sounds good.